City-of-Helsinki/drupal-helfi-platform-config
# UHF-8932: Sidebar content and TPR unit config changes

# [UHF-8932](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8932)


* TPR unit page design has changed. Has affects to the sidebar content.

## What was done


* Removed unit contact information from the sidebar.
* Added latitude and longitude fields to unit display settings.

## How to install
* For full install/testing see other PR's section
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-8932_Unit_layout_changes`
* Run `make drush-updb drush-cr`

## How to test



## Other PRs


* https://github.com/City-of-Helsinki/drupal-hdbt/pull/786


[UHF-8932]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
